### ISSUE TITLE
helm chart docs outdated

### ISSUE THREAD

**Issue by Morriz** [goharbor/harbor#16793]
Hi, your docs state instructions that don't work for helm v3. Please either update the docs to reflect v3 or put the new v3 docs as main, and add a v2 section for reference. I suggest the latter.

Specific problematic parts I encountered:

https://goharbor.io/docs/1.10/working-with-projects/working-with-images/managing-helm-charts/#working-with-the-helm-cli

**Comment by Morriz** [comment]
I use superlatives when expressing myself, but removed them for the sake of argument ;)

**Comment by ChristianCiach** [comment]
You've linked to an ancient versions of the docs (version 1.10). These are the current docs: https://goharbor.io/docs/2.5.0/working-with-projects/working-with-images/managing-helm-charts/

**Comment by Morriz** [comment]
Well, as you can see those docs are exactly the same and outdated @ChristianCiach...

**Comment by nicely-dunn** [comment]
To add on to this, since [Helm 3.7.0](https://github.com/helm/helm/releases/tag/v3.7.0) the command `helm chart` has been removed. Which makes both the documentation and the commands generated by the WebUI outdated for OCI-compatible registries. 

In order to pull from push and pull from my Harbor instance, I had to use the following commands with Helm `v3.7.1`:
```
helm push service-1.5.0.tgz oci://example.org/root/application
```
```
helm pull oci://example.org/root/application/service --version 1.5.0
```

Additionally, as shown in the pull command, the version must explicitly be set instead of appending to the url like `:1.5.0`.
Helm gives the following error otherwise:
```
Error: --version flag is explicitly required for OCI registries
```
